Lib/test/test_importlib/test_resource.py

```diff
@@ -131,10 +131,9 @@ def test_submodule_contents_by_name(self):
 
 
 class NamespaceTest(unittest.TestCase):
-    def test_namespaces_cant_have_resources(self):
-        contents = set(resources.contents(
-            'test.test_importlib.data03.namespace'))
-        self.assertEqual(len(contents), 0)
+    def test_namespaces_cannot_have_resources(self):
+        contents = resources.contents('test.test_importlib.data03.namespace')
+        self.assertEqual(len(set(contents)), 0)
```

Comment:
`self.assertEqual(list(contents), [])` or `self.assertFalse(list(contents))` could provide more informative error report (including the content list itself) if failed.

Why `set()` is used in other tests? Only because the order is not specified? `assertCountEqual()` in addition could check that items are not repeated (`set()` hides duplications).

Reply:
That's a good point.  I don't feel a burning need to change all the tests though.  One thing that would be helpful is to keep in mind that I am cross-porting changes here into `importlib_resources`.  That's not a reason not to use whatever Python 3.7 constructs make sense, just to be mindful that I have to somehow make it work in 2.7, and 3.4-3.6 :)